x/sp/keeper/msg_server.go

```diff
@@ -152,10 +152,7 @@ func (k msgServer) CreateStorageProvider(goCtx context.Context, msg *types.MsgCr
 		FreeReadQuota: msg.FreeReadQuota,
 	}
 	k.SetSpStoragePrice(ctx, spStoragePrice)
-	err = k.UpdateSecondarySpStorePrice(ctx)
-	if err != nil {
-		return nil, err
-	}
+	k.SetSpUpdatePriceTimes(ctx, sp.Id, k.GetSpUpdatePriceTimes(ctx, sp.Id)+1)
```

Comment:
`k.GetSpUpdatePriceTimes(ctx, sp.Id)` seems unnecessary, as it always to be zero here.

Reply:
remove the `times` record - when we set the `update global price interval`, it means the env is dev/qa, we don't have to limit the times to update sp price.